crates/core/src/address/raw.rs

```diff
@@ -65,8 +65,10 @@ pub enum Discriminant {
     IbcToken = 13,
     /// MASP raw address.
     Masp = 14,
+    /// Replay protection
+    ReplayProtection = 15,
     /// Temporary storage address.
-    TempStorage = 15,
+    TempStorage = 16,
```

Comment:
we should avoid doing this. when adding new addrs, append a new entry with the value of the previous entry + 1. this has less chances of breaking existing networks (I think the shielded expedition net is still up)

Reply:
Easy fix, I just went for alphabetic order